syntax = "proto3";

message GamePlayers {
    string player_name = 1;
    int32 age = 2;
    string email = 3;
    string phone_number = 4;
}